### ISSUE TITLE
Camera is super sensitive to motion on Chrome Beta 66 on Android

### ISSUE THREAD

**Issue by HyvelTjuven** [aframevr/aframe#3476]
**Description:**
This issue makes the camera jump around like crazy, being super sensitive to any motion whatsoever.
I assume this issue is related to issue [#3458](https://github.com/aframevr/aframe/issues/3458) but instead of the camera moving slowly it instead moves far too quickly.

Here is a video showing the issue, the video was recorded with the phone held with stable hands.
https://vimeo.com/260723899

- A-Frame Version: 0.8.0
- Platform / Device: Chrome Beta 66 / Oneplus 5 (Android 8.0.0)

**Comment by dmarcos** [comment]
This is because `DeviceMotionEvent.rotationRate` units have changed from degrees per second to radians on Chrome m66

**Comment by jsantell** [comment]
Updating to the latest A-Frame contains an updated webvr-polyfill which fixes these issues -- not sure what A-Frame release contains these changes, though (but introduced after 0.8.0)

**Comment by RaoelHulst** [comment]
Any idea when this will be solved? Thanks in advance.

**Comment by dmarcos** [comment]
@RaoelHulst The fix that @jsantell has mentioned has shipped already. Update A-Frame to 0.8.2

**Comment by dmarcos** [comment]
@jsantell do you think we can close this issue?

**Comment by jsantell** [comment]
yeah, m65 will be replaced with m66 soon and polyfill has been updated with all issues

**Comment by RaoelHulst** [comment]
Thanks man! It works 👍 .

**Comment by a5507203** [comment]
@dmarcos Is there any way to modify web-polyfill to newest version in Aframe 0.5.0? Since I have already built an large AFrame project with 0.5.0, a lot of thing need to change if i swap to new version of aframe. Thanks in advance.

**Comment by ngokevin** [comment]
You can try to build yourself. Clone the repo, checkout v0.5.0 tag, update package.json, `npm run dist`. YMMV

**Comment by nadeemRev** [comment]
I am using A-frame 0.6.1. How can I fix this without updating to 0.8.0.

**Comment by jsantell** [comment]
What @ngokevin said is the only way, using the 0.6.1 tag:

> You can try to build yourself. Clone the repo, checkout v0.5.0 tag, update package.json, npm run dist. YMMV

**Comment by jonwaterschoot** [comment]
I've been away for a while from A-Frame. Wanted to test some of my older experiments, but saw this issue. I went to the homepage of A-Frame and noticed the same issue with some of the examples there.
Using Chrome on a new ZTE Axon 7.

**Comment by dmarcos** [comment]
@jonwaterschoot With A-Frame 0.8.2 and Chrome m66?

**Comment by gabrieljbaker** [comment]
I noticed this on my Samsung s8 + last night. Aframe .8.2 and chrome m66.

**Comment by AlbirrKarim** [comment]
Its same, my device Asus Zenfone max pro m1 Chrome mobile 68 and aframe 8.2  please fix this bug

**Comment by jayson04** [comment]
Hi @ngokevin I am beginner in git. Do you have a video instruction on how to make my own build like what you commented on May 3. Thank you.

**Comment by ClosetMonkey** [comment]
Why is this issue still present on the A-frame demo's at aframe.io? I'm still having this issue on the demos, and when I download and edit aframe version to 0.8.2+ (including 0.9.1) the only change is that gyroscope seems disabled and camera seems stuck in a diagonal view. What is the solid solution to these issues?

Pixel 3, tested using current versions of Chrome and Firefox Focus with the same results

**Comment by dmarcos** [comment]
What Chrome version? What demos? https://aframe.io/aframe/examples/ work fine for me in Chrome m74 and m76 (Canary),

**Comment by tinchoforever** [comment]
Samsung S8 & Chrome m74 here....
All examples.

**Comment by dmarcos** [comment]
Make sure you are on A-Frame 0.9.2. Examples work fine for me on Chrome m74 and Pixel 2 & 3: https://aframe.io/aframe/examples/

**Comment by dmarcos** [comment]
Also make sure you have the devicemotion APIs enabled. More info: https://github.com/aframevr/aframe/issues/3976#issuecomment-494015698

**Comment by marcusx2** [comment]
How do I fix this problem? Sorry, I don't even use aframe but when looking up on google about this problem this is the only lead I found. I tried downloading this project and searching for a fix, but coudln't figure it out. The [Gyroscope api](https://developer.mozilla.org/en-US/docs/Web/API/Gyroscope) works fine but I can't use it because it only works on Android. So I am left with this DeviceMotionEvent.rotationRate, but the rotations are crazy, moving superfast everywhere.

**Comment by jsantell** [comment]
@marcusx2 this chunk of code in [cardboard-vr-display](https://github.com/immersive-web/cardboard-vr-display/blob/master/src/pose-sensor.js) controls the device motion in aframe, maybe a good place to start

**Comment by dmarcos** [comment]
@marcusx2 What is the problem? Do the A-Frame [examples work for you](https://aframe.io/aframe/examples/)? Make sure you are using A-Frame 1.0.4
